items/HOE_OF_NO_TILLING.json

```diff
@@ -0,0 +1,24 @@
+{
+  "itemid": "minecraft:golden_hoe",
+  "displayname": "§aHoe of No Tilling",
+  "nbttag": "{Unbreakable:1b,HideFlags:254,display:{Lore:[0:\"§7Untills farmland!\",1:\"§7\",2:\"§7§8§oMathematically proven to be more\",3:\"§8§oefficient than jumping on it!\",4:\"\",5:\"§7§8This item can be reforged!\",6:\"§a§lUNCOMMON HOE\"],Name:\"§aHoe of No Tilling\"},ExtraAttributes:{date:1727675711532L,price:787878787L,id:\"HOE_OF_NO_TILLING\",bid:18,player:\"§6[MVP§4++§6] LAURIN8§f\",auction:13}}",
```

Comment:
I guess we might want to generalize/remove the auction attributes?

Reply:
Doesnt really matter